ERCS/erc-7580.md

```diff
@@ -0,0 +1,92 @@
+---
+eip: 7580
+title: Inter dapp tracking inferface
+description: Share an inter-dapp tracking inferface for dapp interactions and support for promotion/ads solution
+author: wart (@wartstone)
+discussions-to: https://ethereum-magicians.org/t/erc-7580-inter-dapp-tracking-inferface/17653
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-12-13
+---
+
+## Abstract
+
+This proposal aims to establish an inter-dapp tracking standard inferface for dapp interactions and support for more complicated business logics.
+
+As dapps shows their values and more dapps are coming onto chain,there's plenty of requirements for interactions between dapps. For one like, we have two entities holding their respective contracts: contract A and contract B. Then A targets on those users who did some key moves(commit specific functions) in contract B and would like to give bonus/airdrop to these users. Sure B would also get incentives in the meanwhile. To connect all these dots, B needs to identity these users, verify they're coming for the A's bonus. Hence, we need a track mechanism to facilitate such business.
+
+This track mechanism try to offer a neaty and convenient interface for contracts to interact with each other.
```

Comment:
Your abstract should contain enough substance for a reader to get a high-level technical overview of how your proposal works. After reading this, I vaguely understand that this proposal is to identify users that have called some function and give them a reward from a different contract.

I'd like to see this section reworded a bit to be more understandable, and definitely more detail.